### PR TITLE
fix(generator): delete generouted

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix MANAGER-12814
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Delete generouted

### CODE REVIEW

File: packages/manager-components/src/components/typography/links/links.component.tsx [hunk 1]
```diff
@@ -52,7 +55,7 @@ export const Links: React.FC<LinksProps> = ({
       <OsdsLink
         color={ODS_THEME_COLOR_INTENT.primary}
         download={download}
-        href={href}
+        onClick={() => navigate(href)}
```
[the remark this answers is not in the document]
Comment (aboungnaseng-ovhcloud) [reply packages/manager-components/src/components/typography/links/links.component.tsx:58]: Nice feedback, I change this

File: packages/manager/core/shell-client/src/hooks/useGuideUtils.tsx [hunk 2]
```diff
@@ -100,26 +97,39 @@ type GetGuideLinkProps = {
 function getGuideListLink({ region, subsidiary }: GetGuideLinkProps) {
   const baseUrl = `${baseUrlPrefix?.[region]?.[subsidiary]}`;
   const list: { [guideName: string]: string } = {};
-  for (const [key, value] of Object.entries(GUIDE_LIST)) {
-    list[key] = `${baseUrl}${value[subsidiary]}`;
-  }
+  const tmp = Object.entries(GUIDE_LIST);
+  tmp.forEach((value) => {
```
[the remark this answers is not in the document]
Comment (aboungnaseng-ovhcloud) [reply packages/manager/core/shell-client/src/hooks/useGuideUtils.tsx:101]: 👍

File: packages/manager/core/generator/app/conditional-templates/dashboard/Tab2.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (aboungnaseng-ovhcloud) [reply packages/manager/core/generator/app/conditional-templates/dashboard/Tab2.tsx:1]: Remove the Tab2 exemple ?

File: packages/manager/core/generator/app/conditional-templates/dashboard/_layout.tsx.hbs [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    }
+  }, [location.pathname]);
+
+  const header = {
+    description: 'Description du {{appName}}',
+    title: serviceName,
+  };
+
+  const linkProps = {
+    label: 'Retour à la liste',
```
[the remark this answers is not in the document]
Comment (aboungnaseng-ovhcloud) [reply packages/manager/core/generator/app/conditional-templates/dashboard/_layout.tsx.hbs:67]: yes we can translate it

File: packages/manager/core/shell-client/src/hooks/useBreadcrumb.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState, useContext } from 'react';
+import { useLocation } from 'react-router-dom';
+import { ShellContext } from '../ShellContext';
+
+type BreadcrumbItem = {
+  label: string | undefined;
```
Comment (chipp972) [inline packages/manager/core/shell-client/src/hooks/useBreadcrumb.tsx:6]:
```suggestion
  label?: string;
```

Comment (aboungnaseng-ovhcloud) [reply packages/manager/core/shell-client/src/hooks/useBreadcrumb.tsx]: 👍

**Review by chipp972 — APPROVED** [review]

File: packages/manager/apps/container/src/core/dev/index.ts [hunk 6]
```diff
@@ -26,16 +26,21 @@ export function setupDevApplication(shell: Shell) {
     // if the application configuration doesn't exist in DEV mode
     // then it's a new application, create a default configuration to use
     if (!containerApp) {
-      const devConfig = {
+      let devConfig = {
```
Comment (vovh) [inline packages/manager/apps/container/src/core/dev/index.ts:29]: devConfig is not reassigned, shouldn't we keep const ?

**Review by vovh — CHANGES_REQUESTED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Review by vovh — APPROVED** [review]

**Pull request merged**